### PR TITLE
revise unit definition and remove "value" definition

### PR DESCRIPTION
**Pull request by rsnancollas**

### CODE REVIEW

File: part-spec/definitions.json [hunk 1]
```diff
@@ -396,27 +396,54 @@
       "type": "object",
       "properties": {
         "siUnit": {
-          "description": "name of SI unit of measure",
+          "description": "name of unit",
           "enum": [
+            "microvolt",
+            "milivolt",
```
Comment (asadate) [inline part-spec/definitions.json:402]: typo "millivolt"

File: part-spec/definitions.json [hunk 2]
```diff
@@ -396,27 +396,54 @@
...
             "volt",
+            "kilovolt",
+            "microamp",
+            "miliamp",
```
Comment (asadate) [inline part-spec/definitions.json:406]: typo "milliamp" and all other "milli" in the file

File: part-spec/definitions.json [hunk 3]
```diff
@@ -396,27 +396,54 @@
...
             "coulomb",
+            "milihenry",
             "henry",
+            "picofarad",
+            "nanofarad",
+            "microfarad",
+            "milifarad",
             "farad",
+            "nanosecond",
+            "milisecond",
```
Comment (asadate) [inline part-spec/definitions.json:421]: worth adding "microsecond"

File: part-spec/definitions.json [hunk 4]
```diff
@@ -429,13 +456,10 @@
...
           "type": "number"
         },
-        "unitText": {
-          "description": "human readable text describing value - example 40mV would have a value of mV",
-          "type": "string"
-        },
         "unitFactor": {
-          "description": "multiplier on the value to achieve the SI unit - example for 40mV the unitFactor would be 0.001",
-          "type": "number"
+          "description": "multiplier on the value to achieve the SI unit selected - example for 40mV the unitFactor would be 0.001",
```
Comment (asadate) [inline part-spec/definitions.json:460]: Do we still need this or can we let the tools add this intelligence?

**Review by asadate — CHANGES_REQUESTED** [review]
Additional discussions in the WG.

File: part-spec/definitions.json (hunk 4 above)
Comment (rsnancollas) [reply part-spec/definitions.json:460]: I think Herman still wanted the ability to do this.

**Comment by rsnancollas** [comment]
I'm closing this because the structure changed significantly so I incorporated this feedback in a new PR.

**Pull request closed without merging**